includes/class-wc-payment-token-swedbank-pay.php

```diff
@@ -120,10 +120,10 @@ public function set_recurrence_token( $masked_pan ) {
 	 */
 	public function is_default() {
 		// Mark Method as Checked on "Payment Change" page
-		if ( WC_Gateway_Swedbank_Pay_Cc::wcs_is_payment_change() &&
+		if ( class_exists( 'WC_Subscriptions_Change_Payment_Gateway', false ) &&
+		     WC_Subscriptions_Change_Payment_Gateway::$is_request_to_change_payment &&
 		     isset( $_GET['change_payment_method'] ) &&
-			 abs( $_GET['change_payment_method'] ) > 0 )
-		{
+		     abs( $_GET['change_payment_method'] ) > 0 ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Detected usage of a non-sanitized input variable: $_GET['change_payment_method']](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)